diff -uprN ffcall-1.10/avcall/Makefile.mingw32 ffcall-1.10.mine/avcall/Makefile.mingw32
--- ffcall-1.10/avcall/Makefile.mingw32	2003-10-01 12:46:27.000000000 -0700
+++ ffcall-1.10.mine/avcall/Makefile.mingw32	2016-08-24 11:05:18.822375000 -0700
@@ -72,7 +72,7 @@ minitests.exe : minitests.o $(OBJECTS)
 	$(CC) $(CFLAGS) minitests.o $(OBJECTS) -o minitests.exe
 
 check : all minitests.exe
-	minitests.exe > minitests.out
+	./minitests.exe > minitests.out
 	LC_ALL=C uniq -u < minitests.out > minitests.output.$(HOST)
 # You have to check by hand that minitests.output.$(HOST) is empty.
 
@@ -86,7 +86,7 @@ tests.exe : tests.o $(OBJECTS)
 	$(CC) $(CFLAGS) tests.o $(OBJECTS) -o tests.exe
 
 extracheck : all tests.exe
-	tests.exe > tests.out
+	./tests.exe > tests.out
 	LC_ALL=C uniq -u < tests.out > tests.output.$(HOST)
 # You have to check by hand that tests.output.$(HOST) is empty.
 
diff -uprN ffcall-1.10/callback/Makefile.mingw32 ffcall-1.10.mine/callback/Makefile.mingw32
--- ffcall-1.10/callback/Makefile.mingw32	2003-10-01 12:46:31.000000000 -0700
+++ ffcall-1.10.mine/callback/Makefile.mingw32	2016-08-24 11:05:07.681750000 -0700
@@ -74,7 +74,7 @@ check-subdirs : force
 	$(MAKE) -C trampoline_r -f Makefile.mingw32 check
 
 check : all check-subdirs minitests.exe
-	minitests.exe > minitests.out
+	./minitests.exe > minitests.out
 	LC_ALL=C uniq -u < minitests.out > minitests.output.$(HOST)
 
 tests.o : $(srcdir)/tests.c callback.h vacall_r.h trampoline_r.h
diff -uprN ffcall-1.10/callback/trampoline_r/Makefile.mingw32 ffcall-1.10.mine/callback/trampoline_r/Makefile.mingw32
--- ffcall-1.10/callback/trampoline_r/Makefile.mingw32	2001-08-20 09:18:22.000000000 -0700
+++ ffcall-1.10.mine/callback/trampoline_r/Makefile.mingw32	2016-08-24 11:07:03.103625000 -0700
@@ -69,11 +69,11 @@ test2.exe : test2.o $(OBJECTS)
 	$(CC) $(CFLAGS) test2.o $(OBJECTS) -o test2.exe
 
 check1 : all test1.exe
-	test1.exe
+	./test1.exe
 
 check : all test1.exe test2.exe
-	test1.exe
-	test2.exe
+	./test1.exe
+	./test2.exe
 	touch tests.passed.$(HOST)
 
 extracheck : check
diff -uprN ffcall-1.10/trampoline/Makefile.mingw32 ffcall-1.10.mine/trampoline/Makefile.mingw32
--- ffcall-1.10/trampoline/Makefile.mingw32	2001-08-20 09:18:51.000000000 -0700
+++ ffcall-1.10.mine/trampoline/Makefile.mingw32	2016-08-24 11:04:45.447375000 -0700
@@ -69,11 +69,11 @@ test2.exe : test2.o $(OBJECTS)
 	$(CC) $(CFLAGS) test2.o $(OBJECTS) -o test2.exe
 
 check1 : all test1.exe
-	test1.exe
+	./test1.exe
 
 check : all test1.exe test2.exe
-	test1.exe
-	test2.exe
+	./test1.exe
+	./test2.exe
 	touch tests.passed.$(HOST)
 
 extracheck : check
diff -uprN ffcall-1.10/vacall/Makefile.mingw32 ffcall-1.10.mine/vacall/Makefile.mingw32
--- ffcall-1.10/vacall/Makefile.mingw32	2003-10-01 12:46:34.000000000 -0700
+++ ffcall-1.10.mine/vacall/Makefile.mingw32	2016-08-24 11:06:16.478625000 -0700
@@ -79,7 +79,7 @@ minitests.exe : minitests.o $(OBJECTS)
 	$(CC) $(CFLAGS) minitests.o $(OBJECTS) -o minitests.exe
 
 check : all minitests.exe
-	minitests.exe > minitests.out
+	./minitests.exe > minitests.out
 	LC_ALL=C uniq -u < minitests.out > minitests.output.$(HOST)
 
 tests.o : $(srcdir)/tests.c vacall.h
@@ -92,7 +92,7 @@ tests.exe : tests.o $(OBJECTS)
 	$(CC) $(CFLAGS) tests.o $(OBJECTS) -o tests.exe
 
 extracheck : all tests.exe
-	tests.exe > tests.out
+	./tests.exe > tests.out
 	LC_ALL=C uniq -u < tests.out > tests.output.$(HOST)
 
 mostlyclean : clean
